.pre-commit-config.yaml

```diff
@@ -41,8 +41,9 @@ repos:
     hooks:
       - id: validate-configs-syntax
         name: validate-configs-syntax
-        entry: python3 ./snuba/validate_configs.py
-        language: system
-        pass_filenames: false
+        entry: python3 -m snuba.validate_configs
+        language: python
+        additional_dependencies: [ 'jsonschema', 'pyyaml' ]
+        files: 'snuba/datasets/configuration/*'
```

Comment:
hmmmm this is supposed to trigger a warning that apparently isn't happening -- but anyway here's the warning message it's supposed to print:

```
[WARNING] The 'files' field in hook 'validate-configs-syntax' is a regex, not a glob -- matching '/*' probably isn't what you want here
```

Reply:
How did you generate that warning? When switching to glob, I get the following error:
```An error has occurred: InvalidConfigError:
==> File .pre-commit-config.yaml
==> At Config()
==> At key: repos
==> At Repository(repo='local')
==> At key: hooks
==> At Hook(id='validate-configs-syntax')
==> At key: files
=====> 'snuba/datasets/configuration/**/*{.py,.yaml}' is not a valid python regex
```

Also tried with `snuba/datasets/configuration/**/*`

Reply:
I copied it from the source (it's supposed to show up but current doesn't do to an oversight pre-commit/pre-commit#2521) -- the point is it's *not* a glob, it's a regex -- but you've written it like a glob

for example your pattern currently matches `snuba/datasets/configuration` and `snuba/datasets/configuration//////////////////////////////////////////////////////`